src/Microsoft.PowerApps.TestEngine/Reporting/TestReporter.cs

```diff
@@ -215,12 +215,12 @@ public void StartTest(string testRunId, string testId)
             testRun.ResultSummary.Counters.InProgress++;
         }
 
-        public void SkipTest(string testRunId, string testId)
+        public void FailTest(string testRunId, string testId)
         {
             var testRun = GetTestRun(testRunId);
             var testResult = testRun.Results.UnitTestResults.Where(x => x.TestId == testId).First();
-            testRun.ResultSummary.Counters.NotExecuted++;
-            testResult.Outcome = "NotExecuted";
+            testRun.ResultSummary.Counters.Failed++;
+            testResult.Outcome = "Failed";
```

Comment:
nit: Since this is just a string we're assigning you could consider making this a `const` and then you can refer to it in your tests. Makes it a bit more manageable if we want to change in the future.

Reply:
Good point, ty!. addressed it.